src/3box.js

```diff
@@ -88,20 +88,11 @@ class Box extends BoxApi {
   }
 
   async _load (opts = {}) {
-    const address = await this._3id.getAddress()
-    const { rootStoreAddress, did } = address ? await this._getLinkedData(address) : {}
-    if (rootStoreAddress) {
-      await this.replicator.start(rootStoreAddress, did, { profile: true })
-      await this.replicator.rootstoreSyncDone
-      const authData = await this.replicator.getAuthData()
-      await this._3id.authenticate(opts.spaces, { authData })
-    } else {
-      await this._3id.authenticate(opts.spaces)
-      const rootstoreName = this._3id.muportFingerprint + '.root'
-      const key = (await this._3id.getPublicKeys(null, true)).signingKey
-      await this.replicator.new(rootstoreName, key, this._3id.DID)
-      this._publishRootStore(this.replicator.rootstore.address.toString())
-    }
+    await this._3id.authenticate(opts.spaces)
```

Comment:
How can we authenticate without authData? This would not work for IDW. We need to load toostore and get the auth data in order to send it to the auth function.

Reply:
This was skipped for now, since auth data will be added through ceramic tiles in the upcoming sprint.